armeria-backend/zio/src/main/scala/sttp/client3/armeria/zio/ArmeriaZioBackend.scala

```diff
@@ -34,7 +34,11 @@ private final class ArmeriaZioBackend(runtime: Runtime[Any], client: WebClient,
     }
 
   override protected def streamToPublisher(stream: Stream[Throwable, Byte]): Publisher[HttpData] =
-    runtime.unsafeRun(stream.mapChunks(c => Chunk.single(HttpData.wrap(c.toArray))).toPublisher)
+    Unsafe.unsafeCompat { implicit u =>
+      Runtime.default.unsafe
```

Comment:
Should this be using the `runtime` class variable instead of `Runtime.default`?

Reply:
done